icefall/utils.py

```diff
@@ -975,8 +951,8 @@ def concat(
         ans = k2.ragged.cat([ragged, pad], axis=1)
     else:
         raise ValueError(
-            f'Unsupported direction: {direction}. " \
-            "Expect either "left" or "right"'
+            f'Unsupported direction: {direction}. "             "Expect either "left"'
+            ' or "right"'
```

Comment:
It looks like this string was wrongly mixing single quotes and double quotes, and was formatted weirdly